app/controllers/services_controller.rb

```diff
@@ -22,20 +23,22 @@ def update
 
   def create
     if @bibliography_service.update(update_params)
+      @exhibit.sync_bibliography
       redirect_to edit_exhibit_services_path(@exhibit), notice: I18n.t('services.create.notice')
     else
       redirect_to edit_exhibit_services_path(@exhibit), alert: I18n.t('services.create.error')
     end
   end
 
+  def sync
+    @exhibit.sync_bibliography
+    redirect_to edit_exhibit_services_path(@exhibit), notice: I18n.t('services.sync.started')
+  end
+
   private
 
   def build_resource
-    @bibliography_service = if @exhibit.bibliography_service
-                              @exhibit.bibliography_service
-                            else
-                              ::BibliographyService.new(exhibit_id: @exhibit.id)
-                            end
+    @bibliography_service = BibliographyService.find_or_initialize_by(exhibit_id: @exhibit.id)
```

Comment:
What would you think about changing this to:

```
@bibliography_service = @exhibit.bibliography_service || @exhibit.build_bibliography_service
```

That way, the controller doesn't have to know that a `BibliographyService` class exists.